mlflow/tracking/_model_registry/fluent.py

```diff
@@ -425,6 +427,43 @@ def register_prompt(
     )
 
 
+@require_prompt_registry
+def search_prompts(
+    filter_string: Optional[str] = None,
+    max_results: int = SEARCH_MAX_RESULTS_DEFAULT,
+    page_token: Optional[str] = None,
+) -> PagedList[RegisteredModel]:
```

Comment:
```suggestion
) -> PagedList[Prompt]:
```

Same applied to the docstring.